README.md

```diff
@@ -26,14 +26,8 @@ yarn install
 
 ## Compile contracts
 
-### Prerequisites
-
-- [docker](https://www.docker.com/)
-
-After installation, go to the above `protocol` directory, have Docker running, and:
-
 ```sh
-make
+yarn compile
 ```
 
 ## Test
```

Comment:
Test section still talks about Kovan...

1. remove reference to Kovan
2. add explanation that tests are intended to be run off a mainnet fork 
3. add step to add config pointing to a node (give infura as an example)... this can be part of the section in #2 above
4. get rid of "Local blockchain" header as there aren't multiple options anymore